CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 - Changed connection and player-spawn flow.
 - Updated the available prefabs and textures.
 - Disabled anti-aliasing for mobile to improve performance.
+- Removed the `AndroidWorkerConnector` and `iOSWorkerConnector` and their specific scenes. You can now use the `MobileWorkerConnector` and its `FPS-MobileClientScene` to connect to a mobile device.
```

Comment:
Should have something in the breaking changes about the worker types as well.

Reply:
ah, probably should move that to the breaking changes section actually